### ISSUE TITLE
documentation

### ISSUE THREAD

**Issue by hengxti** [waldheinz/fat32-lib#25]
Hi, 
I was trying for a while to find out how to use the library properly. Could you give a short information for e.g. loading an existing image, navigating throw some folders, create a file ?
I am having trouble to find the "user interface" classes. 

Kind regards

**Comment by waldheinz** [comment]
The library exposes only a carefully chosen set of classes and methods, so everything you see in the [JavaDocs](http://waldheinz.github.io/fat32-lib/apidocs/) is considered public API. A good starting point for exploring the API would be the [`FatFileSystem.read`](http://waldheinz.github.io/fat32-lib/apidocs/de/waldheinz/fs/fat/FatFileSystem.html#read(de.waldheinz.fs.BlockDevice, boolean)) method, which expects you to pass in a [`BlockDevice`](http://waldheinz.github.io/fat32-lib/apidocs/de/waldheinz/fs/BlockDevice.html) and gives back an actual [file system](http://waldheinz.github.io/fat32-lib/apidocs/de/waldheinz/fs/fat/FatFileSystem.html) instance to work with, you can just explore the API docs to find out what can be done.

If you want to start reading an existing image file, the [`FileDisk`](http://waldheinz.github.io/fat32-lib/apidocs/de/waldheinz/fs/util/FileDisk.html#FileDisk(java.io.File, boolean)) constructor is for you.

Basically the documentation for this library is intended for people familiar with JavaDoc, but you'd like to write up something more beginner friendly, I'd happily accept contributions for the project wiki. :-)

**Comment by hengxti** [comment]
thank you for the quick response!

**Comment by hengxti** [comment]
OffTopic: How many hours did you spend to create this Project in total?
